platform/src/common/components/Customise/LocationsContentComponent.jsx

```diff
@@ -110,7 +120,35 @@ const LocationsContentComponent = () => {
           </div>
         )}
       </div>
-      {inputSelect && (
+      {/* <div className='mt-4'>
+        {selectedLocations.length > 0 ? (
+          selectedLocations.map((location) => (
```

Comment:
Just curious about this code you have commented out. Is it reserved for future usage or just for just? @FaithDaka

Reply:
With the way things change frequently, it's likely @Baalmart